src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -33,6 +33,7 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
   const locationBlock = bylineBlocks.find(
     (block: any) => block.type === 'location',
   );
+  const imageBlock = bylineBlocks.find((block: any) => block.type === 'image');
```

Comment:
Any reason we can't be more specific with types in here? We know what block structure to expect for bylines don't we?

Reply:
Yep, we know the block structure for a single byline (not sub-bylines) but we struggled to get any other types to work. Def open to suggestions

Reply:
No worries if you've already tried, would be good to follow up with a PR to work out the types, maybe we could have a go together? I need to get better at typescript! Don't block this PR on it though

Reply:
That'd be great!